README.md

```diff
@@ -1,51 +1,66 @@
+
+Example. For release 1.0 (git tag 1.0), resinhup supports the following device slugs: beaglebone-black, intel-nuc, raspberry-pi, raspberry-pi2 and raspberrypi3. For each slug there is a corresponding Dockerfile: Dockerfile.beaglebone-black, Dockerfile.intel-nuc, Dockerfile.raspberry-pi, Dockerfile.raspberry-pi2 and Dockerfile.raspberrypi3. For each Dockerfile there are docker image pushed to the registry:
 
-## How to use
-Prepare the image on the target by running
-```sh
-$ ./prepare.sh
 ```
-This will stop all the running containers.
-
-### Run container manually
-Run the container manually using:
-```sh
-[host     ]$ rce run -ti --privileged --rm --net=host --volume /:/host resinhup /bin/bash
-[container]$ python3 /app/resinhup.py --config /app/conf/resinhup.conf --debug
-[container]$ exit
-[host     ]$ reboot
+registry.resinstaging.io/resinhup/resinhup-raspberrypi3       1.0  c324b00459f3        2 days ago          241.4 MB
+registry.resinstaging.io/resinhup/resinhup-raspberry-pi2      1.0  4ca1d77c1457        2 days ago          174.3 MB
+registry.resinstaging.io/resinhup/resinhup-raspberry-pi       1.0  4ca1d77c1457        2 days ago          174.3 MB
+registry.resinstaging.io/resinhup/resinhup-intel-nuc          1.0  63cc85875b84        2 days ago          225.3 MB
+registry.resinstaging.io/resinhup/resinhup-beaglebone-black   1.0  7553637ea826        2 days ago          258.3 MB
 ```
 
-### Run container automatically
-Run the container automatically:
-```sh
-[host] $ rce run -ti --privileged --rm --net=host --volume /:/host resinhup
-[host] $ reboot
-```
+The images taged as _latest_ are following the HEAD of master branch.
+
+Hint: there is script helper called [docker-build-and-push.sh](https://github.com/resin-os/resinhup/blob/master/scripts/docker-build-and-push.sh) for pushing images to registry. Check its help.
+
+## How to use
+Use run-resinhup.sh [wrapper](https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh) developed in [meta-resin](https://github.com/resin-os/meta-resin). Check _run-resinhup.sh_ help message for all the configuration you can use. Make sure the _run-resinhup.sh_ along with _update-resin-supervisor_ and _resin-device-progress_ scripts are updated.
+
+Pro Hint: In order to make sure these scripts are updated and run resinhup multiple devices **admins** can use a wrapper on top of _run-resinhup.sh_ called [run-resinhup-ssh.sh](https://github.com/resin-os/meta-resin/blob/master/scripts/resinhup/run-resinhup-ssh.sh). This wrapper is not intended for public use as it requires SSH access to the devices over VPN along with admin permissions for API queries. Check _run-resinhup-ssh.sh_ help message for all the configuration you can use.
```

Comment:
There should also be a comma somewhere in there, the semantics are a bit strange.